abb_irb1200_gazebo/urdf/irb1200_5_90.xacro

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" ?>
+
+<robot name="abb_irb1200_5_90" xmlns:xacro="http://ros.org/wiki/xacro">
+  <xacro:include filename="$(find abb_irb1200_gazebo)/urdf/irb1200_5_90_macro.xacro" />
+  <xacro:include filename="$(find abb_irb1200_support)/urdf/irb1200_5_90_macro.xacro" />
+
+  <!-- get base ABB IRB1200 model -->
+  <xacro:abb_irb1200_5_90 prefix="" />
+
+  <!-- get gazebo add-ons -->
+  <xacro:abb_irb1200_5_90_gazebo_spec prefix="" />
```

Comment:
The IRB 120 Gazebo pkg includes the Gazebo/`ros_control` specific bits in this file (see [abb_irb120_gazebo/irb120_3_58_macro.xacro](https://github.com/ros-industrial/abb_experimental/blob/f8e2bf903c0875b0799521ec383d15b76344c249/abb_irb120_gazebo/urdf/irb120_3_58_macro.xacro), with the `world` link bits being added by the [top-level xacro](https://github.com/ros-industrial/abb_experimental/blob/f8e2bf903c0875b0799521ec383d15b76344c249/abb_irb120_gazebo/urdf/irb120_3_58.xacro)).

Would it make sense to do that here as well?

Did you have a specific reason for separating it all out into the `abb_irb1200_5_90_gazebo_spec` macro?

Reply:
Not really, everything is now in the `_gazebo` macro (removed the `_spec` also, as other robots don't have it)